com_009_material_flow_databases/contents/src/__init__.py

```diff
@@ -0,0 +1,114 @@
+# Import libraries
+import os
+import sys
+import logging
+from datetime import datetime
+
+import pyodbc
+import pandas as pd
+import cartoframes
+
+LOG_LEVEL = logging.INFO
+
+# ODBC Connection details -- these can be pulled out into an odbc.ini file
+ODBC_SOURCE_URL = 'vps348928.ovh.net'
+ODBC_PORT = '5432'
+ODBC_DATABASE = 'mfa'
+ODBC_USER = 'mfa'
+ODBC_PASSWORD = os.environ.get('mfa_db_password')
+
+CONNECTION_STRING = 'DRIVER={};SERVER={};PORT={};DATABASE={};UID={};PWD={}'
+cnxnstr = CONNECTION_STRING.format('{PostgreSQL Unicode}', ODBC_SOURCE_URL, ODBC_PORT, ODBC_DATABASE, ODBC_USER, ODBC_PASSWORD)
+
+# Carto Connection details
+CARTO_USER = os.environ.get('CARTO_WRI_RW_USER')
```

Comment:
same thing here, env variables arent homogenized across script so this will make things fail at the server